documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -3,14 +3,6 @@
 ## Version 9.6.0
 
 ### New features
-* 
+* ReversingLabs Scans - this new feature provides analysis of software packages for file-based malware and provisioning of threat intelligence. 
```

Comment:
I know relatively recently there were talks around being limited to malware detection only, @dterrysynopsys I believe the consensus was that we would continue on with the threat intel naming though. Just wanted to bring that up in case I am remembering wrong.

Reply:
Yes, perhaps it would make sense to just put a period after malware and drop the "and provisioning of threat intelligence" piece.